aim/web/api/tags/views.py

```diff
@@ -84,8 +84,19 @@ async def get_tagged_runs_api(tag_id: str, factory=Depends(object_factory)):
     if not tag:
         raise HTTPException
 
+    from aim.storage.sdk.run import Run
+
+    tag_runs = []
+    for run in tag.runs:
+        aimrocks_run = Run(hashname=run.hashname, read_only=True)
```

Comment:
lets not use aimrocks prefix here. this can be rewritten to:
`for tagged_run in tag.runs:`
`run = Run(....`

Reply:
sure 🙌